### ISSUE TITLE
Not working IP function

### ISSUE THREAD

**Issue by trkadam58** [svenhb/GRBL-Plotter#253]
I have tested IP function with GRBL Advanced it is connecting but GRBL-PLOTTER getting Stuck, also i have tested with IP testing software, same problem facing here also, just for your trial i have added IP testing software link.

https://drive.google.com/file/d/1CcnsDOCXZlRIqeOHTIP1pyxAEbOiQzXq/view?usp=sharing

https://www.youtube.com/watch?v=8Zh50QU39x8

**Comment by svenhb** [comment]
Yes it's connected to Ethernet but not to grbl: I don't see the reset-message, like this:
![image](https://user-images.githubusercontent.com/19973251/151116948-ac063516-cb52-473f-91bf-f15d524b188d.png)

**Comment by trkadam58** [comment]
I have check with GRBL i am getting error as below video Link and image.


https://youtu.be/lUIChDd8JkY
![26 01 2022_21 34 12_REC](https://user-images.githubusercontent.com/96012164/151200592-82484100-9256-4bdb-955a-533f404f73b4.jpg)
![26 01 2022_21 33 39_REC](https://user-images.githubusercontent.com/96012164/151200608-4091939c-4bcf-43da-a544-b0285a3134c3.jpg)

**Comment by svenhb** [comment]
Which controller and firmware are you using?
As written, I tried with ESP32: https://github.com/bdring/Grbl_Esp32
On an ESP32 controller, which uses Telnet

**Comment by trkadam58** [comment]
i am using STM32F446RE in LQFP64 package with GRBL-Advanced firmware

https://github.com/Schildkroet/GRBL-Advanced

**Comment by svenhb** [comment]
Did you follow this: For ETHERNET an additional W5500 Module is required. Then uncomment ETH_IF in Platform.h.?
Does it work with Candle? https://github.com/Schildkroet/Candle2

**Comment by trkadam58** [comment]
i have done required setting for ETHERNET also added W5500 Module, and uncomment ETH_IF in Platform.h, and it is working fine with candle software without any problem
![26 01 2022_22 54 41_REC](https://user-images.githubusercontent.com/96012164/151214774-6977ae7b-6ba1-4345-89ef-9d79975b90c2.jpg)
.

**Comment by svenhb** [comment]
Ok, good to know.
Which output do you get, if you enable "Show report" before connecting and then connect to ethernet?
![image](https://user-images.githubusercontent.com/19973251/151221818-93361afe-14e0-46a4-b6c2-8a63bbce7053.png)

**Comment by trkadam58** [comment]
Before connection i have selected show real time status


![123](https://user-images.githubusercontent.com/96012164/151278717-38569d29-c5ba-4278-94bc-f9f84af88183.jpg)

After connection i have selected show real time status, there is no change.
![27 01 2022_07 32 32_REC](https://user-images.githubusercontent.com/96012164/151278713-3b70840d-62e7-418e-9529-73d639aaaf1d.jpg)
